src/PanelTraits/Columns.php

```diff
@@ -22,7 +24,7 @@ public function setColumns($columns)
         if (is_array($columns) && count($columns)) {
             foreach ($columns as $key => $column) {
                 // if label and other details have been defined in the array
-                if (is_array($column)) {
+                if (is_array($column) || $column instanceof Column) {
```

Comment:
There's something iccky about thyis if and the one below. I suppose it works...though.